modules/core/src/test/scala/com/azavea/stac4s/Generators.scala

```diff
@@ -127,7 +127,7 @@ object Generators {
   )
 
   private def finiteDoubleGen: Gen[Double] =
-    arbitrary[Double].filter((n: Double) => !n.isNaN)
+    arbitrary[Double].filter(java.lang.Double.isFinite)
```

Comment:
@jisantuc how about that?

Reply:
seems promising, but I've been tricked before :crossed_fingers: